src/main/java/racing_car/Record.java

```diff
@@ -0,0 +1,29 @@
+package racing_car;
+
+
+public class Record implements Comparable<Record> {
+    private final String name;
+    private final Integer location;
+
+    public Record(String name, Integer location) {
+        this.name = name;
+        this.location = location;
+    }
+
+    public String getName() {
+        return this.name;
+    }
+
+    public Integer getLocation() {
+        return this.location;
+    }
+
+    public boolean isSameLocation(Record record) {
+        return this.location.equals(record.getLocation());
+    }
+
+    @Override
+    public int compareTo(Record o) {
+        return this.getLocation() - o.getLocation();
+    }
```

Comment:
```suggestion
    public int compareTo(Record o) {
        return Integer.compare(this.location, o.location);
    }
```
만약 비교하고자 하는 값들이 음수라면(병무님이 작성해 주신 현재 로직에서는 그럴일이 없지만요 :) ) 
integer.Max_Value 값을 넘어 32비트가 초과될수 있기때문에 오버플로우가 발생할수도 있습니다.
따라서 정수 값을 비교한다면 다음과 같이 변경하면 좋을 것 같습니다 :)